deps/wrapper.cc

```diff
@@ -69,6 +69,61 @@ std::string ToString(ray::FunctionDescriptor function_descriptor)
     return function_descriptor->ToString();
 }
 
+JuliaGcsClient::JuliaGcsClient(const gcs::GcsClientOptions &options)
+  : options_(options) {}
+
+JuliaGcsClient::JuliaGcsClient(const std::string &gcs_address) {
+  options_ = gcs::GcsClientOptions(gcs_address);
+}
+
+Status JuliaGcsClient::Connect() {
+  io_service_ = std::make_unique<instrumented_io_context>();
+  io_service_thread_ = std::make_unique<std::thread>([this] {
+    std::unique_ptr<boost::asio::io_service::work> work(
+        new boost::asio::io_service::work(*io_service_));
+    io_service_->run();
+  });
+
+  gcs_client_ = std::make_unique<gcs::GcsClient>(options_);
+  return gcs_client_->Connect(*io_service_);
+}
+
+void JuliaGcsClient::Disconnect() {
+  if (io_service_) {
+    io_service_->stop();
+  }
+  if (io_service_thread_ && (io_service_thread_->joinable())) {
+    io_service_thread_->join();
+  }
+  if (gcs_client_) {
+    gcs_client_->Disconnect();
+    gcs_client_.reset();
+  }
+}
```

Comment:
I've been using 4-space indentations in our C code. Do we want to make a choice on that now?

Reply:
4 is fine, I just need to configure my editor